### PR TITLE
WIP: Dev TYPO3 10.x

### PR DESCRIPTION
**Pull request by mcmulman**

### CODE REVIEW

**Comment by mcmulman** [comment]
Feel free to make CodeReview and testing!!!

composer req netresearch/rte-ckeditor-image:dev-TYPO3_10.x

**Comment by zelcode** [comment]
Great job, everything is OK.

**Comment by derhansen** [comment]
I tested this branch and it seems quite stable. Using it in a production project now and will report any problems.

**Comment by chludwig** [comment]
Resolved conflicts in https://github.com/netresearch/t3x-rte_ckeditor_image/pull/101 (hope that was the right way to support this pr)

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Add issues or explore the project on [github](https://github.com/netresearch/t3x
     1. with composer from [packagist](https://packagist.org/packages/netresearch/rte-ckeditor-image)
     
         ```shell
-        composer require netresearch/rte-ckeditor-image
+        composer req netresearch/rte-ckeditor-image:dev-TYPO3_10.x
```
Comment (CybotTM) [inline README.md:19]: should be skipped in merge if possible - even dunno how to do this.

Comment (CybotTM) [inline README.md:19]: doesn't changed the package name to "netresearch/rte_ckeditor_image", or am I wrong?

**Review by CybotTM — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mcmulman) [reply README.md:19]: Right, gonna change it back.

**Review by CybotTM — APPROVED** [review]

**Pull request merged**